### PR TITLE
DOC change and clarify non-text example URLs

### LINKED ISSUE #643: DOC Clarification on non-text examples
At the moment the documentation in `model-repos` [says](https://huggingface.co/docs/hub/model-repos):

> Moreover, you can specify non-text example inputs in the model card metadata. Refer [here](https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/input-examples.md) for a complete list of example input formats for all widget types. For vision & audio widget types, provide example inputs with `src` rather than `text`. 
> 
> For example, allow users to choose from two sample audio files for automatic speech recognition tasks by:
> 
> ```yaml
> widget:
> - src: https://cdn-media.huggingface.co/speech_samples/sample1.flac
>   example_title: Librispeech sample 1
> - src: https://cdn-media.huggingface.co/speech_samples/sample2.flac
>   example_title: Librispeech sample 2
> ```

It's not clear to me whether users can upload their own non-text examples, or if there's an interface where they can go through existing examples. If this is clarified somewhere in the documentation, I think it's not clear to the reader by the point they get to reading this part of the docs.

### PR DESCRIPTION
**Pull request by adrinjalali**
Fixes #643 

This PR clarifies the non-text example URLs in the documentation.

cc @osanseviero @julien-c

### CODE REVIEW

**Review by julien-c — APPROVED** [review]
looks great to me!

**Comment by osanseviero** [comment]
Looks good!

**Comment by adrinjalali** [comment]
I'm not sure what the workflow is here. I'm used to "nobody should merge their own contribution" models. So the question is, should I be merging this PR or is somebody else going to, or are we waiting for more reviews.

**Comment by osanseviero** [comment]
Our way so far is that once you have approvals you're free to merge unless there are open requests :smile:

**Comment by julien-c** [comment]
deployed @adrinjalali: https://huggingface.co/docs/hub/model-repos#how-can-i-control-my-models-widget-example-input

**Pull request merged**
